addons/web/static/tests/views/graph_view_tests.js

```diff
@@ -3619,7 +3619,37 @@ QUnit.module("Views", (hooks) => {
 
         assert.doesNotHaveClass(target, "o_view_sample_data");
         assert.containsOnce(target, ".o_graph_canvas_container canvas");
+        assert.containsOnce(target, ".o_view_nocontent");
+    });
+
+    QUnit.test("empty graph view without sample data after filter", async function (assert) {
```

Comment:
I think the test should be simplified. You can start with a false domain
`domain: Domain.FALSE.toList(),`
and have the two last assertions

Reply:
empty graph view without sample data after filter ->
no content help displayed over empty canvas when no data
?